From fa8fcab1e0d31074c0644c4ac5194474c6c26415 Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Fri, 27 Jul 2012 13:43:33 +0200
Subject: [PATCH 101/278] x86: h264_chromamc_10bit: drop pointless PAVG
 %define

It is only used in one place so there is no need for the abstraction.
---
 libavcodec/x86/h264_chromamc_10bit.asm |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/libavcodec/x86/h264_chromamc_10bit.asm b/libavcodec/x86/h264_chromamc_10bit.asm
index 4481efe..d24308d 100644
--- a/libavcodec/x86/h264_chromamc_10bit.asm
+++ b/libavcodec/x86/h264_chromamc_10bit.asm
@@ -245,7 +245,7 @@ cglobal %1_h264_chroma_mc2_10, 6,7
 %if %0==3
     movq          %2, %3
 %endif
-    PAVG          %1, %2
+    pavgw         %1, %2
 %endmacro
 
 %define CHROMAMC_AVG  NOTHING
@@ -260,7 +260,6 @@ CHROMA_MC4 put
 CHROMA_MC2 put
 
 %define CHROMAMC_AVG  AVG
-%define PAVG          pavgw
 INIT_XMM sse2
 CHROMA_MC8 avg
 %if HAVE_AVX_EXTERNAL
-- 
1.7.9.5

